.buildkite/pipeline.yml

```diff
@@ -217,6 +186,7 @@
             '
 
         echo "+++ Run tests"
+        export JULIA_MPI_TEST_EXCLUDE="test_allreduce.jl,test_reduce.jl,test_scan.jl"
```

Comment:
Need to investigate why these tests fail on AMDGPU backend. Maybe @pxl-th has an idea?